zerver/lib/import_realm.py

```diff
@@ -931,8 +956,8 @@ def import_message_data(import_dir: Path) -> None:
         re_map_foreign_keys(data, 'zerver_usermessage', 'message', related_table="message")
         re_map_foreign_keys(data, 'zerver_usermessage', 'user_profile', related_table="user_profile")
         fix_bitfield_keys(data, 'zerver_usermessage', 'flags')
-        update_model_ids(UserMessage, data, 'usermessage')
-        bulk_import_model(data, UserMessage)
+
+        bulk_import_user_message_data(data)
```

Comment:
Hmm, I'm suspicious of removing the `update_model_ids` call here.

Reply:
That was a mistake!  Thanks for catching that, will be fixed in next push.